tracer.go

```diff
@@ -109,8 +111,10 @@ func NewTracer(
 	zipkinPropagator := &zipkinPropagator{tracer: t}
 	t.addCodec(ZipkinSpanFormat, zipkinPropagator, zipkinPropagator)
 
-	if t.baggageRestrictionManager == nil {
-		t.baggageRestrictionManager = newDefaultBaggageRestrictionManager(&t.metrics, 0)
+	if t.baggageRestrictionManager != nil {
```

Comment:
if the manager isn't passed in, the baggageSetter defaults to use the DefaultManager.